Firestore/core/test/unit/model/value_util_test.cc

```diff
@@ -112,6 +114,25 @@ class ValueUtilTest : public ::testing::Test {
     }
   }
 
+  void VerifyRelaxedAscending(Message<google_firestore_v1_ArrayValue>& left,
```

Comment:
Can you add a method comment here and above?

Reply:
Done.